vnext/Shared/Shared.vcxitems

```diff
@@ -14,13 +15,97 @@
   </ItemGroup>
   <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)AbiSafe.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AsyncStorageModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AsyncStorage\AsyncStorageManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AsyncStorage\FollyDynamicConverter.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)AsyncStorage\KeyValueStorage.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)AsyncStorage\StorageFileIO.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)BaseScriptStoreImpl.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)BatchingMessageQueueThread.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)ChakraRuntimeHolder.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)CreateModules.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)CxxMessageQueue.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)DevServerHelper.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)DevSettings.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)etw\react_native_windows.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)HermesRuntimeHolder.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IDevSupportManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IHttpResource.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)INativeUIManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)InstanceManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IReactRootView.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IRedBoxHandler.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IUIManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IWebSocketResource.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)JSBigAbiString.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)LayoutAnimation.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Logging.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)MemoryMappedBuffer.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)MemoryTracker.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\ExceptionsManagerModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\I18nModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\PlatformConstantsModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\SourceCodeModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\StatusBarManagerModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\UIManagerModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\WebSocketModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)NativeModuleProvider.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)OInstance.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Pch\pch.h" />
```

Comment:
>  <ClInclude Include="$(MSBuildThisFileDirectory)Pch\pch.h" /> [](start = 2, length = 62)

Shared project must not have a pch to avoid interfering with project's pch that includes it.